src/app/main/header/header.component.html

```diff
@@ -11,14 +11,15 @@ <h1 class="mat-headline phone-tablet-only">DSP</h1>
     <span class="fill-remaining-space"></span>
 
     <!-- search-panel (in desktop and tablet version) -->
-    <span class="action">
-        <dsp-search-panel
+   <!--  <span class="action"> -->
+        <!-- button to reactivate when dsp-app will be used as a research platform again: -->
+        <!-- <dsp-search-panel
             [projectfilter]="true"
             [advanced]="true"
             [expert]="true"
             (search)="doSearch($event)">
-        </dsp-search-panel>
-         <!-- advanced and expert search (in desktop and tablet version) -->
+        </dsp-search-panel> -->
+         <!-- to keep? > advanced and expert search (in desktop and tablet version) -->
```

Comment:
"to keep?" --> good point. I think we can delete it. It could be done in a separate refactoring task